index.html

```diff
@@ -1,62 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
   </head>
```

Comment:
hello Elhama Tokhi,
It seems like you have not passed linter errors, here you have a lot of extra lines. Also, you have missed meta.